Olfeo/olfeo-secure-web-gateway/ingest/parser.yml

```diff
@@ -0,0 +1,51 @@
+name: olfeo
```

Comment:
```suggestion
name: olfeo
ignored_values: ["-"]
```

Reply:
It will break the smart description selection:   {
    "value": "{http.request.method} request from unknown user with host {source.ip} on {url.original} (status: {http.response.status_code})",
    "relationships": [
      {
        "source": "source.ip",
        "target": "url.original",
        "type": "requested"
      },
      {
        "source": "url.original",
        "target": "destination.domain",
        "type": "hosted on"
      }
    ],
    "conditions": [
      {
        "field": "source.ip"
      },
      {
        "field": "source.user.name",
        "value": "-"
      }
      
 I can select the best smart description with an other way ?

Reply:
done